code/modules/mob/living/simple_animal/hostile/statue.dm

```diff
@@ -45,12 +45,21 @@
 
 	var/cannot_be_seen = 1
 	var/mob/living/creator = null
+	var/active = 1
+	var/timer = -1
+
+	var/mob/living/victim = null
+	var/intialTox = 0 	//these are here to keep the internal mob from taking damage from things that logically wouldn't affect a rock
+	var/intialFire = 0	//it's a little sloppy I know but it was this or the GODMODE flag. Lesser of two evils.
```

Comment:
Does it take at least two ticks for death to occur - one to take damage, one to die?
I'm seeing the possibility of accumulating radiation, hallucinate, maybe EMPs.